src/balancerd/Cargo.toml

```diff
@@ -10,7 +10,7 @@ publish = false
 workspace = true
 
 [dependencies]
-anyhow = { version = "1.0.95", features = ["backtrace"] }
+anyhow = "1.0.95"
```

Comment:
Not really related to the main change of this PR, just some cleanup. The `backtrace` feature doesn't do anything for Rust versions >= 1.65, just pulls in an unnecessary dependency.